docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+services:
+  lightspeed-ingest:     
+    restart: on-failure
+    build: 
+      context: ../Lightspeed-ingest 
+      dockerfile: Dockerfile
+    image: lightspeed-ingest
+    env_file: '.env'
```

Comment:
I don't have a strong opinion on that, but I feel like it's easier to just have the environment variables and other variables all in docker-compose.yml to have only a single file to manage instead of two. What do you think?

Reply:
Adding environment variables to the docker-compose file gets overly verbose over time. With an environment file it is more organized (I should add sections). My opinion is that you have an environment file which is the only thing a user needs to customize.

Reply:
I get your point. It just feels like another layer of complexity (why specify a port mapping in an env file and not in the ports section of the docker-compose.yml for example).

Except if you don't understand how docker-compose.yml files are structured then it's definitely useful. It could be nice for docker newbies.

Reply:
It's more for newbies, yes.

If we wanted to be verbose (harder to maintain) we could provide defaults that an `.env` file overrides. But that is a bit complex

```
environment:
    STREAM_KEY: "${STREAM_KEY:-}"
ports:
    - "${WEBSOCKET_PORT:8080}:8080"
```



Here's another reason you want an ENV file:
```
command: ["lightspeed-webrtc", "--addr=0.0.0.0", "--ip=${WEBSOCKET_HOST}", "--ports=20000-20500", "run"]
```

Otherwise you are expecting the user to know that they need to set `--ip=` to be identical to `lightspeed-react`'s container environment variable `WEBSOCKET_HOST`.